proxy/dialers/postgres/hook.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2017 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package postgres adds a 'cloudsqlpostgres' driver to use when you want
+// to access a Cloud SQL Database via the go database/sql library.
+// It is a wrapper over the driver found at github.com/lib/pq.
+// To use this driver, you can look at an example in
+// postgres_test package in the hook_test.go file
+package postgres
```

Comment:
I would argue that this way is better because this is exactly how the lib/pq driver is used. 

```
import _ "github.com/lib/pq"
db, err := sql.Open("postgres", "host=ipaddress user=postgres dbname=postgres password=password")
```

But if you insist, I can add the helper functions later on.